valve-monitor/web-app/src/styles/Header.module.scss

```diff
@@ -21,8 +22,8 @@
   }
 }
 
-@media screen and (max-width: $breakpoints-xs) {
+@media screen and (max-width: $breakpoints-md) {
   .header {
-    padding: 0 20px !important;
+    padding: 0 26px !important;
```

Comment:
same here

Reply:
This `:global .ant-xyz` is only supported inside of some sort of a CSS class wrapper, if it's a top level component, the CSS module freaks out. To avoid breaking the rest of the header CSS that hinges on `.header` being the top level CSS class, I'm going to leave as is